### PR TITLE
new license: OGL-UK-3.0

### PR DESCRIPTION
**Pull request by jlovejoy**
test: new license adding xml file for OGL-3.0

### CODE REVIEW

File: src/OGL-UK-3.0.xml [hunk 1]
```diff
@@ -0,0 +1,65 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<SPDXLicenseCollection xmlns="http://www.spdx.org/license">
+    <license isOsiApproved="false" licenseId="OGL-3.0" name="Open Government License v3.0" listVersionAdded="3.3">
```
Comment (swinslow) [inline src/OGL-UK-3.0.xml]: Here too -- "License" should probably be "Licence" given the spelling below.

File: src/OGL-UK-3.0.xml [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+            <p>This is version 3.0 of the Open Government Licence. The National Archives may, from time to time, issue new versions of the Open Government Licence. If you are already using Information under a previous version of the Open Government Licence, the terms of that licence will continue to apply.</p>
+            <p>These terms are compatible with the Creative Commons Attribution License 4.0 and the Open Data Commons Attribution License, both of which license copyright and database rights. This means that when the Information is adapted and licensed under either of those licences, you automatically satisfy the conditions of the OGL when you comply with the other licence. The OGLv3.0 is Open Definition compliant.</p>
+            <p>Further context, best practice and guidance can be found in the UK Government Licensing Framework section on The National Archives website.</p>
+        </text>
+    </license>
+</SPDXLicenseCollection>
```
Comment (swinslow) [inline src/OGL-UK-3.0.xml]: Looks like it's complaining that there's no newline at the end of the file. I don't know whether that matters here...?

File: src/OGL-UK-3.0.xml (hunk 1 above)
Comment (swinslow) [reply src/OGL-UK-3.0.xml]: Also, licenseId is "OGL-3.0" but should be "OGL-UK-3.0"

**Comment by jlovejoy** [comment]
ignore this PR - can't figure out how to separate from old one... created new, clean PR for OGL-UK-3.0 at #712

**Comment by jlovejoy** [comment]
closing now, new PR made.

**Pull request closed without merging**